src/CodeGenHelpers/MethodBuilder.cs

```diff
@@ -22,6 +22,8 @@ internal MethodBuilder(string name, Accessibility? accessModifier, ClassBuilder
             Class = builder;
         }
 
+        public DocumentationComment XmlDoc { get; } = new DocumentationComment();
```

Comment:
this should probably be private and only exposed via the helper methods on the MethodBuilder.